codetf.schema.json

```diff
@@ -210,17 +210,64 @@
           "type": "string",
           "description": "Name of the tool that detected the issue"
         },
-        "ruleId": {
-          "type": "string",
-          "description": "ID of the tool vendor rule that detected the issue"
+        "rule": {
+          "$ref": "#/definitions/detector/rule",
+          "description": "The rule that detected the issue"
         },
-        "ruleDescription": {
-          "type": "string",
-          "description": "Short description of the tool vendor rule that detected the issue"
+        "findings": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/detector/finding" }
         }
       },
       "additionalProperties": true,
-      "required": ["name", "ruleId"]
+      "required": ["name", "rule", "findings"]
+    },
+
+    "detector": {
+      "finding": {
+        "type": "object",
+        "properties": {
+          "guid": {
+            "type": "string",
+            "description": "A unique identifier for the finding"
+          },
+          "fixed": {
+            "type": "boolean",
+            "description": "Whether the finding was fixed by the codemod"
+          },
+          "reason": {
+            "type": "string",
+            "description": "Reason the finding was not fixed"
+          }
+        },
+        "additionalProperties": true,
```

Comment:
I like this section, but it could be potentially _massive_. Scanners commonly find pathological conditions where they end up reporting 50,000 things. Is there anything we can/should do in the spec to protect ourselves from massive, redundant info?

Reply:
We can add a `maxItems` although it's incumbent upon the consumer to enforce that. Basically it would mean we'd ignore any results beyond `maxItems`.